src/main/java/org/folio/marccat/dao/CatalogDAO.java

```diff
@@ -110,7 +121,8 @@ public void updateBibNote(final int amicusNumber, final int noteNumber, final Se
     } catch (SQLException ex) {
       throw new SQLException(ex);
```

Comment:
What purpose does catching, wrapping (in another instance of SQLException), and rethrowing this exception serve?

Reply:
I have removed the catch